src/components/Car/carForm/carForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { saveCar } from '../../../store/car';
+import './carForm.css';
+
+const AddCar = () => {
```

Comment:
- [ ] I still can't add cars.  This happens when I click the submit button in the form:

### Firefox:

![image](https://user-images.githubusercontent.com/57421823/207817150-91288006-11fe-4839-981c-c7be4e0bbb57.png)

### Chrome

![image](https://user-images.githubusercontent.com/57421823/207817249-b73b6fdc-39c8-4d88-97dc-7b86ec586713.png)

### Rails Console

![image](https://user-images.githubusercontent.com/57421823/207817459-36cf8d64-334e-4926-81b5-fde47c895fe3.png)

It seems that params[:email] is nil, so it is not possible for the server to get the user id.

Kindly fix this bug so your app works as intended 👍